inst/private/python_ipc_pytave.m

```diff
@@ -70,7 +70,6 @@
 
   s = strjoin(cmd, newl);
   pyexec(s)
-  
-  info.raw = pyeval('str(_outs)');
```

Comment:
Oh ok. Actually I still don't know how this is used. I probably haven't
looked much into it.